src/components/ReportsPage/HeatmapPage/HeatmapUtils.ts

```diff
@@ -59,33 +61,53 @@ export const getSortedRulesByLevelsFromScores = (
     ),
   );
 
-type GroupByKeys = 'teams' | 'tags' | 'ladderLevels';
+type GroupByKeys = 'teams' | 'tags' | 'ladderLevels' | 'domains';
```

Comment:
What about the "Entity" groupBy? Is that intentionally missing omitted here? If so, why?

Reply:
Ah. Maybe worth a code comment indicating as much to prevent others from having similar confusion.